bugapp/Create.go

```diff
@@ -61,7 +61,11 @@ func Create(Args ArgumentList) {
 
 	var mode os.FileMode
 	mode = 0775
-	os.Mkdir(string(dir), mode)
+	err := os.Mkdir(string(dir), mode)
+	if err != nil {
```

Comment:
This would be a little cleaner as `if err := os.Mkdir(string(dir), mode); err != nil {`